.github/workflows/builds.yml

```diff
@@ -285,11 +283,9 @@ jobs:
           restore-keys: |
             ${{ runner.os  }}-ccache-linux-nodeps_${{ env.CCACHE_KEY_SUFFIX }}_
 
-
       - name: Configure
-        run: >
+        run: > # disabling ninja because of race condition
```

Comment:
Why is this needed?

Reply:
I don't remember exactly but I think I was facing flapping here because some downloads were too slow/fast

Reply:
let me try to revert the CI changes that are not necessary since master seems to be stable